java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/ShowInputBoxParams.java

```diff
@@ -83,6 +93,22 @@ public String getValue() {
     public void setValue(@NonNull final String value) {
         this.value = Preconditions.checkNotNull(value, "value");
     }
+    
+    /**
+     * Controls if a password input is shown. Password input hides the typed text.
+     */
+    @Pure
+    @NonNull
+    public boolean isPassword() {
+        return password;
+    }
+
+    /**
+     * Controls if a password input is shown. Password input hides the typed text.
+     */
+    public void setPassword(boolean password) {
+        this.password = password;
```

Comment:
Expanding the NetBeans LSP protocol extensions. Brave!

Ideally we should version the changes, but it is a gray area so far. Consider at least increasing spec version of the `java.lsp.server` module and leaving a `@since` note on the new methods/field.